openedx/core/djangoapps/schedules/tests/test_content_highlights.py

```diff
@@ -21,7 +21,8 @@ def setUp(self):
         self._setup_user()
 
     def _setup_course(self):
-        self.course = CourseFactory.create()
+        self.course = CourseFactory.create(
+            highlights_enabled_for_messaging=True)
```

Comment:
Nit: Our convention is to put the ending paren on a new line when the code is separated into multiple lines.